### PR TITLE
SASS -> Sass

### PR DESCRIPTION
**Pull request by jpdevries**
http://sassnotsass.com

See Also FortAwesome/Font-Awesome#8170

### CODE REVIEW

File: src/3.2.1/whats-new/index.html [hunk 1]
```diff
@@ -186,7 +186,7 @@
     </div>
     <div class="span4">
       <h4><i class="icon-terminal"></i> SCSS Support</h4>
-      A long term solution is now in place for SCSS support. Need SASS? Try <a href="http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html#syntax">sass-convert</a>.
+      A long term solution is now in place for SCSS support. Need Sass? Try <a href="http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax">sass-convert</a>.
```
Comment (tagliala) [inline src/3.2.1/whats-new/index.html]: Interesting enough, `http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax` is giving a 404 :)

Please use  `SASS_REFERENCE` here, as it was before

Comment (tagliala) [inline src/3.2.1/whats-new/index.html]: Interesting enough, `http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax` is giving a 404 error here :)

Please use SASS_REFERENCE, as before

**Comment by live789** [comment]
如何退订

在 2017-06-30 19:34:47，"Geremia Taglialatela" <notifications@github.com> 写道：


@tagliala requested changes on this pull request.

In src/3.2.1/whats-new/index.html:

> @@ -186,7 +186,7 @@
     </div>
     <div class="span4">
       <h4><i class="icon-terminal"></i> SCSS Support</h4>
-      A long term solution is now in place for SCSS support. Need SASS? Try <a href="http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html#syntax">sass-convert</a>.
+      A long term solution is now in place for SCSS support. Need Sass? Try <a href="http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax">sass-convert</a>.


Interesting enough, http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax is giving a 404 error here :)

Please use SASS_REFERENCE, as before

**Comment by live789** [comment]
如何退订





--
发自我的网易邮箱手机智能版



在 2017-06-30 19:34:47，"Geremia Taglialatela" <notifications@github.com> 写道：


@tagliala requested changes on this pull request.

In src/3.2.1/whats-new/index.html:

> @@ -186,7 +186,7 @@
     </div>
     <div class="span4">
       <h4><i class="icon-terminal"></i> SCSS Support</h4>
-      A long term solution is now in place for SCSS support. Need SASS? Try <a href="http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html#syntax">sass-convert</a>.
+      A long term solution is now in place for SCSS support. Need Sass? Try <a href="http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax">sass-convert</a>.


Interesting enough, http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax is giving a 404 error here :)

Please use SASS_REFERENCE, as before

**Comment by tagliala** [comment]
@live789 

Go to https://github.com/FortAwesome/Font-Awesome/pull/11243

Click "Unsubscribe" on the right sidebar

image

**Comment by robmadole** [comment]
First off, thanks for spending the time to post this PR. It's sat in the Font Awesome GitHub repo for far too long and we should have responded or addressed the work that you put into this.

Over the past year we've been hard at work on Font Awesome 5. It's now released and this PR no longer applies. So we're going to close it (with an apology for being so tardy).

If you think the work from this still applies feel free to reply and we'll take a second look.

– FA team

**Pull request closed without merging**